cookiecutter/main.py

```diff
@@ -52,7 +52,8 @@ def expand_abbreviations(template, config_dict):
     return template
 
 
-def cookiecutter(template, checkout=None, no_input=False, extra_context=None):
+def cookiecutter(template, checkout=None, no_input=False,
```

Comment:
There might still be external code depending on the position of arguments. Quite unfortunate as Python 2.x doesn't have keyword-only arguments.